arviz/tests/base_tests/test_data.py

```diff
@@ -584,6 +584,99 @@ def test_repr_html(self):
         assert escape(idata.__repr__()) in html
         xr.set_options(display_style=display_style)
 
+    def test_add_groups(self):
+        data = np.random.normal(size=(4, 500, 8))
+        idata = from_dict(
+            posterior={"a": data[..., 0], "b": data},
+            sample_stats={"a": data[..., 0], "b": data},
+            observed_data={"b": data[0, 0, :]},
+            posterior_predictive={"a": data[..., 0], "b": data},
+        )
```

Comment:
can you use the fixture from the other PR to simplify and reduce duplicated code? You can use the fixture and (for `extend`) create `idata2 = create_...` to get a 2nd idata which is different from the fixture

Reply:
Yeah, I will. I was waiting for other PR to merge and then forget altogether.